lints/lint_dnsname_wildcard_left_of_public_suffix.go

```diff
@@ -0,0 +1,54 @@
+package lints
+
+import (
+	"github.com/weppos/publicsuffix-go/publicsuffix"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type DNSNameWildcardLeftofPublicSuffix struct {
+	// Internal data here
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) Initialize() error {
+	return nil
+}
+
+func (l *DNSNameWildcardLeftofPublicSuffix) CheckApplies(c *x509.Certificate) bool {
+	return util.IsSubscriberCert(c)
+}
+
+func wildcardLeftOfPublicSuffix(domain string) bool {
+	parsedDomain, err := publicsuffix.Parse(domain)
+	if err != nil {
```

Comment:
This seems like it should be a fatal. I'm a bit concerned we're handling these inconsistently across lints.